engine/hunspell_table.py

```diff
@@ -6625,6 +6635,22 @@ def _command_toggle_emoji_prediction(self) -> bool:
         self.toggle_emoji_prediction_mode()
         return True
 
+    def _command_trigger_emoji_predictions(self) -> bool:
+        '''Handle hotkey for the command “trigger_emoji_predictions”
+
+        :return: True if the key was completely handled, False if not.
+        '''
+        self._temporary_emoji_predictions = True
+        return True
+
+    def _command_trigger_word_predictions(self) -> bool:
+        '''Handle hotkey for the command “trigger_word_predictions”
+
+        :return: True if the key was completely handled, False if not.
+        '''
+        self._temporary_word_predictions = True
+        return True
+
```

Comment:
`_handle_hotkeys()` looks like it can only call one of these at a time even if they both have the same hotkey. Personally, I'd want to give them different hotkeys, but I could picture somebody wanting to trigger both at the same time. Do you think I should add a way to do that? Maybe a third hotkey, `trigger_all_predictions`?

Reply:
I think `_handle_hotkeys()` can call several things if they share the same hotkey. I am pretty sure about that because `Tab` is usually bound both to `enable_lookup` and `select_next_candidate` and I remember seing both being called when pressing Tab recently when I worked on the `ja-anthy` support. So I think it works but should verify this again.

Reply:
Ah, no, I think it can call several commands as long as they keep failing (I.e. returning False) but it stops at the first command which succeeds.

Reply:
For now, do you think it's ok to leave this requiring separate hotkeys for `trigger_emoji_predictions` and `trigger_word_predictions`? That works well for me, and if somebody wants to use the same hotkey for them, I don't think changing it later would be too hard.